tests/e2e/framework/main_entry.go

```diff
@@ -78,6 +179,40 @@ func (f *Framework) tearDown() error {
 		return err
 	}
 
+	log.Printf("cleaning up Profile Bundles")
+	err = f.cleanUpProfileBundle("rhcos4")
```

Comment:
I am wondering should we remove all profile bundle at this point?

Reply:
Yeah - I wondered about that, too. When I ran tests prior to implementing this refactor, these were the only profile bundles left (which I think makes sense since they're the only ones bootstrapped with the environment).

I decided to only tear down what's created. But perhaps that is a case to take a more aggressive cleanup approach?